__tests__/fixtures/responses/swagger.json

```diff
@@ -0,0 +1,56 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "0.0.1",
+        "title": "your title"
+    },
+    "paths": {
+        "/persons": {
+            "get": {
+                "operationId": "get_person",
+                "description": "Gets `Person` object.",
+                "responses": {
+                    "200": { "description": "empty schema" },
+                    "201": {
+                        "description": "inline schema",
+                        "schema": { "type": "object", "properties": { "inline_schema": { "type": "string" } } }
+                    },
+                    "202": {
+                        "description": "inline schema ref",
+                        "schema": { "$ref": "#/definitions/direct_schema_ref" }
+                    },
+                    "203": { "$ref": "#/responses/empty_schema" },
+                    "204": { "$ref": "#/responses/regular_schema" },
+                    "205": { "$ref": "#/responses/indirect_schema" },
+                    "206": { "$ref": "#/responses/same_name" },
+                    "207": { "$ref": "#/responses/same_name_clash" }
```

Comment:
response codes `200` to `207` should enumerate all possible combinations with respect to `response` schema definitions